core/src/main/scala/akka/persistence/dynamodb/internal/QueryDao.scala

```diff
@@ -333,6 +337,20 @@ import software.amazon.awssdk.services.dynamodb.model.QueryResponse
   }
 
   def loadEvent(persistenceId: String, seqNr: Long, includePayload: Boolean): Future[Option[SerializedJournalItem]] = {
+    val queryResult = queryForEvent(persistenceId, seqNr, includePayload)
+
+    queryResult.flatMap { maybeItem =>
+      maybeItem match {
+        case None       => queryResult
+        case Some(item) => followBreadcrumb(item).map(Some.apply)(ExecutionContext.parasitic)
+      }
+    }(ExecutionContext.parasitic)
```

Comment:
seems dangerous to use that much parasitic when it's not obvious that nothing will throw